aqua/misc.aqua

```diff
@@ -0,0 +1,72 @@
+import "@fluencelabs/aqua-lib/builtin.aqua"
```

Comment:
I think it makes sense to add a little explanation (comment) on what this file is, why it is named `misc`, whether it's a public API or not and why.

Reply:
good comment!